app/models/bot/slack.rb

```diff
@@ -25,23 +25,21 @@ def should_notify?(target, model)
   end
 
   def notify_slack(model)
-    p = self.get_project(model)
-    t = self.get_team(model, p)
+    t = self.get_team(model)
```

Comment:
if we have `AnnotationBase.team`, we can remove all other methods.

Reply:
Yes I added to `AnnotationBase.team` and removed all other methods